docs/how_it_works/univariate_drift_detection.rst

```diff
@@ -108,6 +108,36 @@ Wasserstein distance between the two distributions is given by:
 .. math::
     W_1\left(X_i^{ref},X_i^{ana}\right) = \int_\mathbb{R}\left|\hat{F}_{ref}(x)-\hat{F}_{ana}(x)\right|dx
 
+.. _univariate-drift-detection-cont-hellinger:
+
+Hellinger Distance
+........................
+
+The Hellinger Distance, is a distance metric used to quantify the similarity between two distributions. It measures the overlap between the probabilities assigned 
+to the same event by both reference and analysis. It ranges from 0 to 1 where a value of 1 is only achieved when reference assigns zero probability to each event to which 
```

Comment:
It would be good to say that reference and analysis are samples. Also - and this likely applies for other methods as well - I m not too keen on naming them reference and analysis. Because while we may be using the whole reference dataset on one side, we are only using a chunk on the other. Speaking about analysis here may confuse people.

Reply:
Damn, good point. Maybe in how it works where, we describe all the methods, we should just name them p and q, or sample A and B, or whatever. Apart from that we should have a paragraph that explains how this works in the library, that reference set serves as sample A, and every chunk of data that is inputed to _calculate/estimate_ method is the B sample?